cmd/flow_display.go

```diff
@@ -0,0 +1,279 @@
+	writeBuf("Capture size: %s\n", sizestr.ToString(totalBytes))
+	if len(strings.TrimSpace(options)) > 0 {
+		writeBuf("Options: %s\n", options)
+	}
+	if strings.Contains(options, "background=true") {
+		writeBuf("Showing last: %d\n", flowsToShow)
+		writeBuf("Display: %s\n", display.getCurrentItem().name)
+		writeBuf("Enrichment: %s\n", enrichment.getCurrentItem().name)
+	} else {
+		writeBuf("Showing last: %d Use Up / Down keyboard arrows to increase / decrease limit\n", flowsToShow)
+		writeBuf("Display: %s Use Left / Right keyboard arrows to cycle views\n", display.getCurrentItem().name)
+		writeBuf("Enrichment: %s Use Page Up / Page Down keyboard keys to cycle enrichment scopes\n", enrichment.getCurrentItem().name)
+	}
+
+	if display.getCurrentItem().name == rawDisplay {
+		outputBuffer.WriteString("Raw flow logs:\n")
+		for _, flow := range lastFlowsFiltered {
+			writeBuf("%v\n", flow)
+		}
+		writeBuf("%s\n", strings.Repeat("-", 500))
+	} else {
+		// recreate table from scratch
+		headerFmt := color.New(color.BgHiBlue, color.Bold).SprintfFunc()
+		columnFmt := color.New(color.FgHiYellow).SprintfFunc()
+
+		// main field, always show the end time
+		colIDs := []string{
+			"EndTime",
+		}
+
+		// enrichment fields
+		if enrichment.getCurrentItem().name != noOptions {
+			colIDs = append(colIDs, enrichment.getCurrentItem().ids...)
+		} else {
+			// TODO: add a new flag in the config to identify these as default non enriched fields
+			colIDs = append(colIDs,
+				"SrcAddr",
+				"SrcPort",
+				"DstAddr",
+				"DstPort",
+			)
+		}
+
+		// append interfaces and their directions between enrichment and features
+		// this is useful for pkt drop, udns etc
+		colIDs = append(colIDs,
+			"Interfaces",
+			"IfDirections",
+		)
+
+		// standard / feature fields
+		if display.getCurrentItem().name != standardDisplay {
+			for _, col := range cfg.Columns {
+				if col.Field != "" && slices.Contains(display.getCurrentItem().ids, col.Feature) {
+					colIDs = append(colIDs, col.ID)
+				}
+			}
+		} else {
+			// TODO: add a new flag in the config to identify these as default feature fields
+			colIDs = append(colIDs,
+				"FlowDirection",
+				"Proto",
+				"Dscp",
+				"Bytes",
+				"Packets",
+			)
+		}
+
+		colInterfaces := make([]interface{}, len(colIDs))
+		for i, id := range colIDs {
+			colInterfaces[i] = ToTableColName(id)
+		}
+		tbl := table.New(colInterfaces...)
+		tbl.WithWriter(outputBuffer)
+		tbl.WithHeaderFormatter(headerFmt).WithFirstColumnFormatter(columnFmt)
+
+		// append most recent rows
+		for _, flow := range lastFlowsFiltered {
+			tbl.AddRow(ToTableRow(flow, colIDs)...)
+		}
+
+		// inserting empty row ensure minimum column sizes
+		emptyRow := []interface{}{}
+		for _, id := range colIDs {
+			emptyRow = append(emptyRow, strings.Repeat("-", ToTableColWidth(id)))
+		}
+		tbl.AddRow(emptyRow...)
+
+		// print table
+		tbl.Print()
+	}
+
+	if len(keyboardError) > 0 {
+		writeBuf(keyboardError)
```

Comment:
I see a linter error here: 
```
printf: non-constant format string in call to github.com/netobserv/network-observability-cli/cmd.writeBuf (govet)go-golangci-lint
```

I think you could do 2 versions of `writeBuf` that would follow the usual pattern: `writeBuf` takes only a string argument, and `writeBuff` takes varargs

Reply:
I don't see it running make lint :thinking: 

Are you using `golangci-lint@v1.64.6` ?